### ISSUE TITLE
The preview window does not appear for "CocFzfList diagnostics"

### ISSUE THREAD

**Issue by sethfowler** [antoinemadec/coc-fzf#21]
I'm not getting a preview window when I run `CocFzfList diagnostics`. Hitting "?" just types a "?" normally into the fuzzy finder; it doesn't make a preview window appear.

I have `junegunn/fzf.vim` installed, and I do get a preview window when I use its commands, so I don't think I have any kind of global FZF configuration that's disabling it.

**Comment by antoinemadec** [comment]
Could you share a minimal vimrc to reproduce this issue? Also could you run :checkhealth and copy the output?

**Comment by sethfowler** [comment]
@antoinemadec I actually added some debug logging to `coc-fzf` and tracked this down. The source of the issue is this check:
```viml
let g:coc_fzf_preview_available = 1
try
  call fzf#vim#with_preview()
catch
  let g:coc_fzf_preview_available = 0
endtry
```

If `fzf.vim` and `coc-fzf` load in this order:
```viml
Plug 'junegunn/fzf.vim'
Plug 'antoinemadec/coc-fzf'
```

Then that check works, and I get the preview window.

If they load in this order:
```viml
Plug 'antoinemadec/coc-fzf'
Plug 'junegunn/fzf.vim'
```

Then the check fails, so the preview window gets disabled.

Since I have my `Plug` entries listed in alphabetical order, I was hitting the second case, and the preview window wasn't showing up.

I've fixed this locally by swapping the order of the two `Plug` entries. May be worth either documenting this or making that check lazily so that the order doesn't matter.
